pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -68,6 +68,9 @@ type EventListenerTrigger struct {
 	// +optional
 	// TODO(dibyom): Allow multiple interceptors
 	Interceptor *EventInterceptor `json:"interceptor,omitempty"`
+
+	// TODO(dibyom): Remove this before 0.3 release
```

Comment:
instead of TODO(name) i think its better in the long run to open an issue (or refer to an existing issue) e.g. TODO(#11111) - i think we'll have a better chance of remembering if there's an issue and less of a chance of stale TODOs floating around (we'll be able to tell when they're done - and we can even add automation if we want!)

Reply:
Excellent idea!